### PR TITLE
Add a description testing blah blah

### PR DESCRIPTION
**Pull request by RMacfarlane**
- [ ] foo
- [ ] bar

### CODE REVIEW

**Comment by rebornix** [comment]
update version
```diff

```

**Comment by rebornix** [comment]
Upper case.
```diff

```

**Comment by rebornix** [comment]
update uppercase
```diff

```

**Comment by rebornix** [comment]
```diff
diff --git a/package.json b/package.json
index 98945a7..369ef2f 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
-  "description": "Shopping cart utility",
+  "description": "Shopping Cart Utility",
   "dependencies": {
     "body-parser": "~1.17.1",
     "cookie-parser": "~1.4.3",

```

**Comment by rebornix** [comment]
upper
```diff

```

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
+  "description": "Shopping cart utility",
   "dependencies": {
     "body-parser": "~1.17.1",
```
[the remark this answers is not in the document]
Comment (RMacfarlane) [reply package.json]: b

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
+  "description": "Shopping cart utility",
```
Comment (RMacfarlane) [inline package.json]: asdf

**Comment by RMacfarlane** [comment]
```diff
diff --git a/package.json b/package.json
index 98945a71..bfadd771 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,6 @@
   "email": "gtsopour@gmail.com",
   "private": false,
   "scripts": {
-    "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
   "description": "Shopping cart utility",

```

**Comment by RMacfarlane** [comment]
```diff
diff --git a/package.json b/package.json
index 98945a71..81bba818 100644
--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
-  "private": false,
+  "private": true,
   "scripts": {
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"

```

**Comment by RMacfarlane** [comment]
```diff
diff --git a/package.json b/package.json
index 98945a71..81bba818 100644
--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
-  "private": false,
+  "private": true,
   "scripts": {
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"

```

**Comment by RMacfarlane** [comment]
```diff
diff --git a/package.json b/package.json
index 98945a71..50ce6eff 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
+  "foo": "bar",
   "private": false,
   "scripts": {
     "start": "node ./bin/www",

```

**Comment by RMacfarlane** [comment]
asdfasdf```diff
diff --git a/package.json b/package.json
index 98945a71..0d23dc1f 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
+  "asdf": "asfasdf",
   "private": false,
   "scripts": {
     "start": "node ./bin/www",

```

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RMacfarlane) [reply package.json]: b

[the remark this answers is not in the document]
Comment (RMacfarlane) [reply package.json]: b

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
```
Comment (RMacfarlane) [inline package.json]: asdf

File: package.json [hunk 4]
```diff
@@ -8,6 +8,7 @@
...
+  "description": "Shopping cart utility",
   "dependencies": {
     "body-parser": "~1.17.1",
     "cookie-parser": "~1.4.3",
```
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 4 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json [hunk 5]
```diff
@@ -8,6 +8,7 @@
     "start": "node ./bin/www",
```
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 1 above)
Comment (RMacfarlane) [inline package.json]: asdf

**Comment by RMacfarlane** [comment]
asdf

File: package.json [hunk 6]
```diff
@@ -8,6 +8,7 @@
...
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
   },
+  "description": "Shopping cart utility",
   "dependencies": {
```
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 1 above)
Comment (RMacfarlane) [inline package.json]: asdf

File: package.json [hunk 7]
```diff
@@ -8,6 +8,7 @@
     "start": "node ./bin/www",
     "start_azure_debug": "if [[ ${APPSVC_TUNNEL_PORT} != \"\" ]]; then node --inspect=0.0.0.0:$APPSVC_TUNNEL_PORT ./bin/www; else npm run start; fi"
```
Comment (RMacfarlane) [inline package.json]: new pending review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: new review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: hajsdhf

File: package.json (hunk 4 above)
Comment (RMacfarlane) [inline package.json]: jljkl

File: package.json (hunk 1 above)
Comment (RMacfarlane) [inline package.json]: jlklkj

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: done

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: 1

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: ?

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: review

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: great

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: testing

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: laksd

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: ajsldkf

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: why

Comment (RMacfarlane) [inline package.json]: why

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: nice

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: done

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: test again

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [reply package.json]: double test

Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [inline package.json]: delete

Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [inline package.json]: asdf

Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: review?

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: end review

Comment (RMacfarlane) [inline package.json]: still pending

File: package.json (hunk 4 above)
Comment (RMacfarlane) [inline package.json]: done

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: add comment

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: part of a pending review

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: finishing this pending review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

Comment (RMacfarlane) [inline package.json]: start review

Comment (RMacfarlane) [inline package.json]: test

Comment (RMacfarlane) [reply package.json]: reply works

Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: done

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: done

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: lasjfl

File: package.json (hunk 7 above)
Comment (RMacfarlane) [inline package.json]: asjdfasklfd

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: alsjkdflkasjdf

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: kll

File: package.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (RMacfarlane) [reply package.json]: ?

Comment (RMacfarlane) [inline package.json]: start review

File: package.json (hunk 3 above)
Comment (RMacfarlane) [inline package.json]: finish review

File: package.json (hunk 2 above)
Comment (RMacfarlane) [inline package.json]: test

File: package.json (hunk 1 above)
Comment (RMacfarlane) [inline package.json]: start

File: package.json (hunk 6 above)
Comment (RMacfarlane) [inline package.json]: start

File: package.json (hunk 1 above)
Comment (RMacfarlane) [inline package.json]: add

File: package.json (hunk 5 above)
Comment (RMacfarlane) [inline package.json]: test

**Pull request merged**